nucleus/validate/client.py

```diff
@@ -175,3 +179,30 @@ def evaluate_model_on_scenario_tests(
             f"validate/{model_id}/evaluate",
         )
         return AsyncJob.from_json(response, self.connection)
+
+    def create_custom_eval_function(
```

Comment:
I think this naming is not clear enough for the intended functionality. We're allowing customer to define some sort of sink that they can funnel their own external metrics into.

We already have the notion of a custom written eval function by customers although it is not exposed. We need to make it much clearer that this is a placeholder evaluation function to externally provide data.